### PR TITLE
fix(scripts): report testName in error

### PR DESCRIPTION
**Pull request by millotp**
## 🧭 What and Why

Add the name of the CTS test in the error message to ease debugging.
The diff is misleading, the only change is the indentation.

### CODE REVIEW

File: generators/src/main/java/com/algolia/codegen/exceptions/CTSException.java [hunk 1]
```diff
@@ -20,4 +26,17 @@ public CTSException(String message, boolean skipable) {
...
   }
+
+  public void setTestName(String testName) {
+    this.testName = testName;
+  }
+
+  @Override
+  public String getMessage() {
+    if (testName != null) {
+      return "Error in " + testName + ": " + super.getMessage();
```
Comment (shortcuts) [inline generators/src/main/java/com/algolia/codegen/exceptions/CTSException.java:37]: wdyt of newlines or decorated before the message? it's pretty verbose sometimes

**Review by shortcuts — COMMENTED** [review]
looks great!!

File: generators/src/main/java/com/algolia/codegen/exceptions/CTSException.java (hunk 1 above)
Comment (millotp) [reply generators/src/main/java/com/algolia/codegen/exceptions/CTSException.java:37]: yep good idea

**Review by shortcuts — APPROVED** [review]
greatooo

**Review by Fluf22 — APPROVED** [review]
👍🏻

**Pull request merged**